src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.ProductDto;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/product")
+public class ProductController {
```

Comment:
Tej klasy nie powinno być w tym Pull Requescie

Reply:
Tą klasę wrzucam tutaj ze względu na to, że dopiero w tym pull requescie przerzucam wszystko do poszczególnych pakietów.